src/lib/components/Segment.svelte

```diff
@@ -114,10 +129,20 @@
       -1px,
       0
     );
-    width: calc((100% - (var(--segments) * var(--padding))) / var(--segments));
-    padding: var(--padding-2x) 0 var(--padding);
+
+    // Calculate the width of the indicator:
+    // (100% - [left and right segment paddings] + [all gaps between buttons]) / [number of segments]
+    width: calc(
+      (
```

Comment:
I still don't understand what's going on with the indentation here.
I've added the current indentation in front of every line here:
```
 6      (
10          100% -
12            (
14              2 * var(--segment-padding) + var(--segment-gap) *
16                (var(--segments) - 1)
12            )
 8        ) / var(--segments)
```
Should the parenthesis with 6 indentation be aligned with the one with 8 indentation?

Also, I think the line breaks could be more intuitive to indicate the priority of addition vs. multiplication.
Would this work?
```
      (
        100% -
          2 * var(--segment-padding) +
          var(--segment-gap) * (var(--segments) - 1)
      ) / var(--segments)
```